### ISSUE TITLE
PHP 8 error in html/ops/dbinfo.php

### ISSUE THREAD

**Issue by gchilders** [BOINC/boinc#5457]
html/ops/dbinfo.php uses each() on line 206. This function is no longer available in PHP 8. The while loop should be replaced with foreach().

Also, I get the following warning at line 214:
Deprecated: number_format(): Passing null to parameter #<span>1 ($num) of type float is deprecated in /home/boincadm/projects/nfs/html/ops/dbinfo.php on line 214

**Comment by AenBleidd** [comment]
@davidpanderson, could you please take a look?

**Comment by AenBleidd** [comment]
@davidpanderson, I might be wrong, but looks like this deprecation was not fixed in #5459
```
Deprecated: number_format(): Passing null to parameter #1 ($num) of type float is deprecated in /home/boincadm/projects/nfs/html/ops/dbinfo.php on line 214
```
Could you please double check?
Thank you in advance.

**Comment by davidpanderson** [comment]
I think it's fixed.  The problem was that the 'SHOW TABLE STATUS' query
returns info both for tables (which have size info)
and views (for which the size info is missing, i.e. null).
My fix skips views, so we don't call number_format() with null.
